config/ip_allowlist.yml

```diff
@@ -5,4 +5,4 @@ test:
   addresses: "127.0.0.1"
 
 production:
-  addresses: <%= ENV["BIBDATA_IP_WHITELIST"] || "" %>
+  addresses: <%= ENV["BIBDATA_IP_ALLOWLIST"] || ENV["BIBDATA_IP_WHITELIST"] || "" %>
```

Comment:
let's look at it with https://github.com/pulibrary/princeton_ansible/pull/1628/files so that we dont need the || "BIBDATA_IP_WHITELIST". Thanks.